spec/systems/users_spec.rb

```diff
@@ -0,0 +1,162 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Signup', type: :system do
+  context 'validates screen' do
+    it 'displays signup screen should show correct field' do
+      visit signup_path
+
+      expect(page).to have_content('Signup')
+
+      within 'form' do
+        expect(page).to have_field('user[username]')
+        expect(page).to have_field('user[password]')
+        expect(page).to have_field('user[password_confirmation]')
+      end
+    end
+
+    it 'displays signup screen show correct label' do
+      visit signup_path
+
+      expect(page).to have_content('Signup')
+
+      within 'form' do
+        expect(page).to have_field('user[username]')
+        expect(page).to have_field('user[password]')
+        expect(page).to have_field('user[password_confirmation]')
+      end
+    end
+  end
+
+  context 'given valid data' do
+    it 'redirects to index page and show flash message' do
+      visit signup_path
+
+      within 'form' do
+        fill_in('user[username]', with: 'nimblehq')
+        fill_in('user[password]', with: 'password')
+        fill_in('user[password_confirmation]', with: 'password')
+
+        click_button('Signup')
```

Comment:
Instead of hard code as `Signup` we could use I18n here `I18n.t('auth.signup')`

Reply:
Fixed in https://github.com/gutakk/ruby-google-scraper/pull/4/commits/99878e0d33ea843326f0431fd14f5d2c27587d57